SWLOR.Game.Server/Service/RoleplayService.cs

```diff
@@ -97,13 +97,13 @@ private static void ProcessPlayerRoleplayXP(NWPlayer player)
             var dbPlayer = DataService.Player.GetByID(player.GlobalID);
             if (dbPlayer.RoleplayPoints >= 50)
             {
-                const int BaseXP = 500;
+                const int BaseXP = 1000;
```

Comment:
Why does this need to be doubled?

Reply:
Felt the RPXP system wasn't giving at that much in retrospect to what you can get by grinding. Basically 1 kill every 30 minutes via RPXP seems low.

Reply:
I'm ok with this.